server/api/search/google.js

```diff
@@ -0,0 +1,67 @@
+const {google} = require('googleapis');
+const credentials = require('../../../secrets.js').googleSheets;
+
+const auth = new google.auth.JWT(
```

Comment:
we discussed offline, but lets wraps these guys in functions like `getGoogleOptions`